### PR TITLE
DBZ-7304 Update MongoDb doc for snapshot modes alignment

### PR DESCRIPTION
**Pull request by mfvitale**
related to: https://github.com/debezium/debezium/pull/5371

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/mongodb.adoc [hunk 1]
```diff
@@ -1911,15 +1960,45 @@ That is, the specified expression is matched against the entire name string of t
...
+Set the `snapshot.mode.custom.name` configuration property to the name provided by the `name()` method of your implementation.
+
+For more information, see xref:connector-custom-snapshot[custom snapshotter SPI].
+endif::community[]
+
+ifdef::community[]
+|[[mongodb-property-snapshot-mode-custom-name]]<<mongodb-property-snapshot-mode-custom-name, `+snapshot.mode.custom.name+`>>
+|No default
+| When `snapshot.mode` is set as `custom`, use this setting to specify the name of the custom implementation provided in the `name()` method that is defined by the 'io.debezium.spi.snapshot.Snapshotter' interface.
+The provided implementation is called after a connector restart to determine whether to perform a snapshot.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/mongodb.adoc:1999]:
```suggestion
|If `snapshot.mode` is set to `custom`, use this setting to specify the name of the custom class that implements the `name()` method that is defined in the 'io.debezium.spi.snapshot.Snapshotter' interface.
After a connector restart, {prodname} calls the specified custom implementation to determine whether to perform a snapshot.
```

File: documentation/modules/ROOT/pages/connectors/db2.adoc [hunk 2]
```diff
@@ -2717,6 +2717,14 @@ Set the `snapshot.mode.custom.name` configuration property to the name provided
 For more information, see xref:connector-custom-snapshot[custom snapshotter SPI].
 endif::community[]
 
+ifdef::community[]
+|[[db2-property-snapshot-mode-custom-name]]<<db2-property-snapshot-mode-custom-name, `+snapshot.mode.custom.name+`>>
+|No default
+| When `snapshot.mode` is set as `custom`, use this setting to specify the name of the custom implementation provided in the `name()` method that is defined by the 'io.debezium.spi.snapshot.Snapshotter' interface.
+The provided implementation is called after a connector restart to determine whether to perform a snapshot.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/db2.adoc:2724]:
```suggestion
| If `snapshot.mode` is set to `custom`, use this setting to specify the name of the custom class that implements the `name()` method that is defined in the 'io.debezium.spi.snapshot.Snapshotter' interface.
After a connector restart, {prodname} calls the specified custom implementation to determine whether to perform a snapshot.
```

File: documentation/modules/ROOT/pages/connectors/mysql.adoc [hunk 3]
```diff
@@ -3451,6 +3451,14 @@ ifdef::community[]
 `custom`:: The connector performs a snapshot according to the implementation specified by the xref:mysql-property-snapshot-mode-custom-name[`snapshot.mode.custom.name`] property, which defines a custom implementation of the `io.debezium.spi.snapshot.Snapshotter` interface.
 endif::community[]
 
+ifdef::community[]
+|[[mysql-property-snapshot-mode-custom-name]]<<mysql-property-snapshot-mode-custom-name, `+snapshot.mode.custom.name+`>>
+|No default
+| When `snapshot.mode` is set as `custom`, use this setting to specify the name of the custom implementation provided in the `name()` method that is defined by the 'io.debezium.spi.snapshot.Snapshotter' interface.
+The provided implementation is called after a connector restart to determine whether to perform a snapshot.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/mysql.adoc:3458]:
```suggestion
| If `snapshot.mode` is set to `custom`, use this setting to specify the name of the custom class that implements the `name()` method that is defined in the 'io.debezium.spi.snapshot.Snapshotter' interface.
After a connector restart, {prodname} calls the specified custom implementation to determine whether to perform a snapshot.
```

File: documentation/modules/ROOT/pages/connectors/oracle.adoc [hunk 4]
```diff
@@ -3165,6 +3165,14 @@ endif::community[]
 
 For more information, see the xref:oracle-connector-snapshot-mode-options[table of `snapshot.mode` options].
 
+ifdef::community[]
+|[[oracle-property-snapshot-mode-custom-name]]<<oracle-property-snapshot-mode-custom-name, `+snapshot.mode.custom.name+`>>
+|No default
+| When `snapshot.mode` is set as `custom`, use this setting to specify the name of the custom implementation provided in the `name()` method that is defined by the 'io.debezium.spi.snapshot.Snapshotter' interface.
+The provided implementation is called after a connector restart to determine whether to perform a snapshot.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/oracle.adoc:3172]:
```suggestion
| If `snapshot.mode` is set to `custom`, use this setting to specify the name of the custom class that implements the `name()` method that is defined in the 'io.debezium.spi.snapshot.Snapshotter' interface.
After a connector restart, {prodname} calls the specified custom implementation to determine whether to perform a snapshot.
```

File: documentation/modules/ROOT/pages/connectors/sqlserver.adoc [hunk 5]
```diff
@@ -2879,6 +2879,14 @@ Set the `snapshot.mode.custom.name` configuration property to the name provided
 For more information, see xref:connector-custom-snapshot[custom snapshotter SPI].
 endif::community[]
 
+ifdef::community[]
+|[[sqlserver-property-snapshot-mode-custom-name]]<<sqlserver-property-snapshot-mode-custom-name, `+snapshot.mode.custom.name+`>>
+|No default
+| When `snapshot.mode` is set as `custom`, use this setting to specify the name of the custom implementation provided in the `name()` method that is defined by the 'io.debezium.spi.snapshot.Snapshotter' interface.
+The provided implementation is called after a connector restart to determine whether to perform a snapshot.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/sqlserver.adoc:2886]:
```suggestion
| If `snapshot.mode` is set to `custom`, use this setting to specify the name of the custom class that implements the `name()` method that is defined in the 'io.debezium.spi.snapshot.Snapshotter' interface.
After a connector restart, {prodname} calls the specified custom implementation to determine whether to perform a snapshot.
```

File: documentation/modules/ROOT/pages/connectors/db2.adoc (hunk 2 above)
Comment (mfvitale) [reply documentation/modules/ROOT/pages/connectors/db2.adoc:2724]: @roldanbob The value of this property is the value provided in the `name()` method of the custom implementation, not the name of the custom class.

Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/db2.adoc:2724]: > @roldanbob The value of this property is the value provided in the `name()` method of the custom implementation, not the name of the custom class.

Ah! Ok, got it. Thanks! Updated comments. Is this better?

**Comment by mfvitale** [comment]
@roldanbob changes addressed.

**Review by roldanbob — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**